pkg/securitycontextconstraints/util/util.go

```diff
@@ -176,3 +176,41 @@ func EqualStringSlices(a, b []string) bool {
 	}
 	return true
 }
+
+// IsOnlyServiceAccountTokenSources returns true if the sources of the projected volume
+// source match to what would be injected by the ServiceAccount volume projection controller
+//
+// This function is derived from pkg/security/podsecuritypolicy/util/util.go with the
+// addition of OpenShift-specific "openshift-service-ca.crt" ConfigMap source.
+func IsOnlyServiceAccountTokenSources(v *api.ProjectedVolumeSource) bool {
+	for _, s := range v.Sources {
+		// reject any projected source that does not match any of our expected source types
+		if s.ServiceAccountToken == nil && s.ConfigMap == nil && s.DownwardAPI == nil {
+			return false
+		}
+		if t := s.ServiceAccountToken; t != nil && (t.Path != "token" || t.Audience != "") {
+			return false
+		}
+
+		if s.ConfigMap != nil {
+			switch cmRef := s.ConfigMap.LocalObjectReference.Name; cmRef {
+			case "kube-root-ca.crt":
```

Comment:
`case "kube-root-ca.crt", "openshift-service-ca.crt":`. Now you fall through to line 207.

Reply:
added a check specific to `kube-root-ca.crt`